LuckParser/Controllers/HTMLBuilder.cs

```diff
@@ -1737,42 +1737,37 @@ private void CreatePlayerTab(StreamWriter sw, int phaseIndex)
                     {
                         sw.Write("<div class=\"tab-pane fade show active\" id=\"home" + pid + "\">");
                         {
-                            List<Tuple<Boon,long>> consume = p.GetConsumablesList(_log, phase.Start, phase.End);
-                            List<Tuple<Boon, long>> initial = consume.Where(x => x.Item2 == 0).ToList();
-                            List<Tuple<Boon, long>> refreshed = consume.Where(x => x.Item2 > 0).ToList();
+                            List<Tuple<Boon, long, int>> consume = p.GetConsumablesList(_log, phase.Start, phase.End);
+                            List<Tuple<Boon, long, int>> initial = consume.Where(x => x.Item2 == 0).ToList();
+                            List<Tuple<Boon, long, int>> refreshed = consume.Where(x => x.Item2 > 0).ToList();
+                            refreshed.Sort((x, y) => x.Item2.CompareTo(y.Item2));
```

Comment:
Why not sort in SetConsumables in Player.cs? So that it's only done once

Reply:
That actually makes more sense indeed, let me change it.